rpcd/etc/openstack_deploy/user_apt_packages.yml

```diff
@@ -0,0 +1,11 @@
+# This file will contain any apt packages that the deployer would like to pin to a particular version.
```

Comment:
:+1: Putting these into the user_variables file will likely give it more exposure. Putting it into a separate file is likely to leave it forgotten.